### ISSUE TITLE
About index

### ISSUE THREAD

**Issue by linzexi2001** [zhaohaoyu376/WI-LD2ND#1]
May I ask if the HU value of CT used in calculating SSIM and PSNR indicators in the paper is between 0-2500

**Comment by Maxwell-Zhao** [comment]
Yes, as shown in our dataloader, we clip the image into 0-2500 for Mayo2016, and -1024-3000 for Mayo2020, respectively.
